rpi-dp/leaveprob-dttor/network_testing.py

```diff
@@ -0,0 +1,57 @@
+# network testing
+# based on this article on pyimagesearch
+# https://www.pyimagesearch.com/2017/12/11/image-classification-with-keras-and-deep-learning/
+
+# To test the network from terminal, make sure you are at rpi-dp/planttp-clfier/, if not cd there and run:
```

Comment:
Change `rpi-dp/planttp-clfier/` to `rpi-dp/leaveprob-dttor/`

Reply:
ok